### PR TITLE
Add new entry to the section for `Bean Validation 2.0 (JSR 380)`.

### PR DESCRIPTION
**Pull request by Naoghuman**
Hi there,

like I said here my PR for a new entry for the `Additional constraints` (Bean Validation 2.0) section to my library Lib-Validation.

Its possible to have a look on the grammatic (I'm not a english native speaker :) )?
Any suggestion for a better wording?

Greetings
Naoghuman

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ so that we can add them here.
  * [Java Bean Validation Extension](https://github.com/nomemory/java-bean-validation-extension) offers a set of useful constraints (`@Alphanumeric`, `@IPv6`, `@StartsWith`...).
  * [The validation project of Markus Malkusch](https://github.com/malkusch/validation) provides another set of constraints (`@BitcoinAddress`, `@ISBN`...).
  * [A nice example of password validator](https://github.com/Baeldung/spring-security-registration/blob/master/src/main/java/org/baeldung/validation/PasswordConstraintValidator.java) based on [Passay](http://www.passay.org/), a password policy enforcement library.
+ * The library [Lib-Validation](https://github.com/Naoghuman/lib-validation) from Naoghuman (Peter Rogge) is developed with the goal to allowed easily `validating` in a JavaFX application. See for example [@NewDuration](https://github.com/Naoghuman/lib-validation/blob/master/src/main/java/com/github/naoghuman/lib/validation/core/annotation/NewDuration.java).
```
Comment (gunnarmorling) [inline resources.md]: If the library is doing more than just defining constraints, you might say:

> The library [Lib-Validation](https://github.com/Naoghuman/lib-validation) from Peter Rogge is developed with the goal to provide validation functionality for JavaFX applications. It also provides custom constraints, e.g. [@NewDuration](https://github.com/Naoghuman/lib-validation/blob/master/src/main/java/com/github/naoghuman/lib/validation/core/annotation/NewDuration.java).

**Review by gunnarmorling — COMMENTED** [review]
No worries, most of us are no English native speakers either :) I've made one rewording suggestion inline.

File: resources.md (hunk 1 above)
Comment (Naoghuman) [reply resources.md]: That's fine. I was also not sure if I should use my 'nickname' from my GitHub account, but your suggestion sound good (I think I will use it for my project description also :) ).

So I do must now a new PR for your suggestion?

Comment (gsmet) [reply resources.md]: I wouldn't use the nickname if I were you, just add a link on your name to your GitHub profile.

You can make your changes directly in this PR, amend your commit and force push your branch.

HTH.

Comment (gunnarmorling) [reply resources.md]: No, you just can push another commit with this change to this PR. I'll then squash the two commits into a single one and merge it.

**Comment by Naoghuman** [comment]
I add a new commit with the new for wording.

**Comment by gunnarmorling** [comment]
Squashed and applied. Should be on the live website soon. Thanks, @Naoghuman!

**Pull request merged**